_component_design/chips.md

```diff
@@ -17,27 +17,87 @@ helix-ui-javascript: false
 
 <section class="static-section" markdown="1">
 
-## Introduction
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-12 hxCol-xs-12 hxCol-sm-12 hxCol-md-12 hxCol-lg-12" %}
+
+## When to use
+
+Use chips to graphically represent a unit of information. For example, use chips to represent a user or device.
+
+## Best practices
+
+Consider the following best practices when constructing a chip:
+
+- Avoid using chips in multiple locations on a page.
+- Label the chip so that the user can identify the information represented by the chip.
+
+{% endcolumn %}
+
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
 
 <div class="hxRow" markdown="1">
 
 {% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-### When to use
 
-Chips are best used with indicating a list of items that may be part of a different workflow. For example, adding or removing a list of requestors on a ticket.
+## Specifications
+Refer to the following specifications when constructing chips.
 
-### Best practices
+{% endcolumn %}
 
-- Avoid using chips in multiple locations on a page.
-- Ensure the user can concretely identify the item represented by the chip.
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Composition
+A chip is composed of a container and an identifying element, known as the **identifier**. The identifier can consist of the following:
```

Comment:
s/`the following`/`the following elements` (see the last row in the table at http://rackerlabs.github.io/docs-rackspace/style-guide/style/lists.html#introductory-text)